content/blog/vs_code_extension_blog.md

```diff
@@ -0,0 +1,52 @@
+
+<div align="center">
+
+  
+
+![Alt text](https://fossoverflow.dev/nextimg/%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png/828/75?url=%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png&w=828&q=75)
+
+  
+
+</div>
+
+<br/>
+I wanted a project which had a tech stack I am comfortable in as well as contain something new to learn, 
+
+so I applied for building a [VS Code Extension](https://github.com/OpenLake/Activity-Tracker/tree/main/packages/vscode-extension) as it was based on MERN stack (on which I have worked before) also it gave me a good chance to learn how can we create our own extensions in vs code.
+
+In today's competitive world everyone wants to be productive in day- to- day life. For that one of the key things is time management and organizing the tasks, especially the programmers like me who want to contribute to different projects. For them it is important to see which projects they are spending more time on and which they are lacking to give time in order to give all the projects the time which it requires and also prioritize them. The VS Code Extension which I build tracks the time which the user spends in various projects in the vs code editor. The Extension graphically shows the data about the projects which makes it easier to interpret the data.
+
+The extension not only stores the time spent but also gets the data about the time spent in writing codes in different languages. So that the user can see which language they use the most and in which they are more efficient.
+
+This Extension is a subset of the activity tracker application which tracks time spent in various applications not only vs code. It is a full stack application which allowed me to learn and understand the various aspects of developing a full stack application. Now I am more confident in developing a scalable full stack application while maintaining a clean and organized code.
+
+During this project I also got to learn Figma for UI design. My first design for the front-end of Activity Tracker is as follows:- 
+
+![](https://lh6.googleusercontent.com/OhVUSmrawchiNkEnZGF2ZQwaIRuWbuc4iA3oZNYGisx9LvGbQfqvVshCorGdlkr4WiywFVl9WcX8dRJYRmnXmKbnbuRBMTEJrJOsol-B5LcdDEvA0oGXjfwqbS-YTcbuC1PXT85_)
+<br/>
+
+The event gave me an opportunity to meet new people and together fix errors in code, designing UI etc. Overall a good experience sharing knowledge with each other.
+
+  
+Special thanks to my mentors [Ambar Mutha](mailto:ambarm@iitbhilai.ac.in), [Kumar Shivendu](mailto:shivendu@iitbhilai.ac.in) and Shikhar Vashistha who were there to help me with my doubts and clearing the errors in code making it more clean and efficient.
```

Comment:
Please don't make emails public 😢. You can use my GitHub.

Reply:
Sorry about that, added the github profile instead.